### PR TITLE
[DNM] [r-mr1] New BSP fixes

### PR DESCRIPTION
**Pull request by ix5**
Plus general cleanup

**DNM** until we've verified the necessity of some calls.

@MarijnS95 get in here ;)

### CODE REVIEW

File: system_ext/private/init.te [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Extension policy for AOSP /system init to cope with
+# our own /odm/system_ext -> (/system)/system_ext symlinks
+
+# Read and mount symlinked files
+alow init vendor_firmware_file:filesystem r_file_perms;
```
Comment (jerpelea) [inline system_ext/private/init.te:5]: typo

Comment (ix5) [reply system_ext/private/init.te]: fixed

**Comment by ix5** [comment]
Still TODO:

```
avc: denied { dac_override } for comm="iptables" capability=1 scontext=u:r:netutils_wrapper:s0 tcontext=u:r:netutils_wrapper:s0 tclass=capability
avc: denied { dac_override } for comm="dpmd" capability=1 scontext=u:r:dpmd:s0 tcontext=u:r:dpmd:s0 tclass=capability
avc: denied { setuid } for comm="dpmd" capability=7 scontext=u:r:dpmd:s0 tcontext=u:r:dpmd:s0 tclass=capability
avc: denied { dac_override } for comm="iptables" capability=1 scontext=u:r:netutils_wrapper:s0 tcontext=u:r:netutils_wrapper:s0 tclass=capability
```

**Comment by MarijnS95** [comment]
> @MarijnS95 get in here ;)

I guess that's a no.

**Pull request merged**